pkg/api/query/querypb/query.proto

```diff
@@ -41,6 +41,7 @@ message QueryRequest {
 
   bool enableDedup = 7;
   bool enablePartialResponse = 8;
+  // Deprecated
   bool enableQueryPushdown = 9;
```

Comment:
I think the convention is to mark a field as reserved instead of a comment. So, it should be:
`reserved 9;`

Reply:
TIL about `reserved`! Thanks